readme.md

```diff
@@ -77,5 +77,5 @@ You're ready to go! Visit Ping CRM in your browser, and login with:
 To run the Ping CRM tests, run:
 
 ```
-phpunit
+phpunit --testsuite=Feature
```

Comment:
an alternative may be to just use full path to phpunit and point to tests directory

```
 ./vendor/bin/phpunit tests